modules/cxf/src/main/java/org/mule/module/cxf/transport/MuleUniversalConduit.java

```diff
@@ -345,6 +350,18 @@ private void sendResultBackToCxf(Message m, MuleEvent resEvent) throws Transform
                 clearClientContextIfNeeded(getMessageObserver());
                 return;
             }
+            // If this branch is reached, it means that the message payload is empty.
+            // By asserting that the inboundProperty 'http.status' exists, I'm verifying that
+            // the last inbound endpoint used was HTTP, and that it set the property correctly.
+            else if (resEvent.getMessage().getInboundPropertyNames().contains("http.status"))
+            {
+                int httpResponseStatusCode = Integer.valueOf(resEvent.getMessage().getInboundProperty("http.status").toString());
```

Comment:
Is there a constant for http.status? Use that in case it exists. static method if possible.

Reply:
I couldn't found a static property in a class worth importing. I moved the field name to static field.